### DISCUSSION TITLE
Driving Cameras lights from Hot end and bed outputs.

### DISCUSSION THREAD

**Discussion by bonz099** [xpDIY/3DPlacer#18] [Ideas]
For those of us who don't plan on switching this back to a 3d printer, you can use the hot end output and bed output to drive a 12v/24v output like lights for the top and bottom cameras. Use a multimeter to determine polarity. Then use this example to setup your triggers. 
![image](https://github.com/xpDIY/3DPlacer/assets/77824691/82edd863-bc0c-4861-8573-b135edea2323)

In our case (BED) "M104 S0" for the false/off state and "M104 S100" for the true/on state.  Use M104 for the hot end output. 

Notes: I had to plug in 10k ntc to the bed temp location to bypass the temp safety feature. You may need to do that for the hot end as well. Im sure a 10k resistor across the points would be fine too.

**Comment by xpDIY** [comment]
Good idea, I am actually thinking driving another air pump, and this could be a good candidate for that purpose as well, thanks for sharing!
